app/models/mentor/request_comment.rb

```diff
@@ -30,7 +30,7 @@ def self.from(request)
       by_student: true,
       content_markdown: request.comment_markdown,
       content_html: request.comment_html,
-      updated_at: request.updated_at,
+      updated_at: request.created_at,
```

Comment:
The problem was that once a mentor request is accepted, its status column is updated which causes the `updated_at` column to be updated too. I've replaced this with the `created_at` column. Note that the returned field needs to be `updated_at` because this is basically faking an actual request comment, and for those we _do_ want the updated date,